src/vs/workbench/browser/parts/views/viewPane.ts

```diff
@@ -545,7 +547,17 @@ export abstract class ViewPane extends Pane implements IView {
 		}
 
 		this.iconContainerHover = this._register(this.hoverService.setupUpdatableHover(getDefaultHoverDelegate('mouse'), this.iconContainer, calculatedTitle));
-		this.iconContainer.setAttribute('aria-label', calculatedTitle);
+		this.iconContainer.setAttribute('aria-label', this._getAriaLabel(calculatedTitle));
+	}
+
+	private _getAriaLabel(title: string): string {
+		const viewHasAccessibilityHelpContent = this.viewDescriptorService.getViewDescriptorById(this.id)?.accessibilityHelpContent;
```

Comment:
Having the `ViewPane` check the storage service to learn whether the accessible view is showing feels like implementation of accessible view leaking into here. Can the the accessible view service have an `isShowing` method?

Reply:
View pane cannot access the layer where the accessible view is. I tried moving the accessible view and it's not possible AFAICT

Reply:
I could set a context key though instead of using the storage service

Reply:
Could we have an `AccessibleViewService` that lives in `src\vs\workbench\services`?

Reply:
I was thinking more that the entirety of this class could move into `vs\workbench\services`, but I'm not going to block the PR on that:

https://github.com/microsoft/vscode/blob/0ba604cc9d59e63419f3dadf0a29b683ccf55c08/src/vs/workbench/contrib/accessibility/browser/accessibleView.ts#L832-L833